submissions/xT0PERx/js-dom/index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="ru">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" type="text/css" href="visuality.css" />
+    <title>My interactive side-menu</title>
+  </head>
+  <body>
+    <div class="container clearfix">
+      <div class="menu">
+        <div class="menu-title"><p>О компании :</p>
+        <ul class="menu-list">
+          <li class="menu-item-1">AUDI</li>
```

Comment:
Avoid using uppercase in HTML markup. Content can be changed but visual representation cannot change.

Reply:
Fixed